code/src/java/pcgen/gui2/tools/Utility.java

```diff
@@ -323,10 +306,6 @@ public static void selectDefaultBrowser(Component parent)
 
 		if (PCGenSettings.getBrowserPath() == null)
 		{
-			//No action, as we have no idea what a good default would be...
```

Comment:
This logic is incorrect, since the IF test was not reversed when only the else was used